src/main/java/com/avairebot/commands/fun/RandomDogCommand.java

```diff
@@ -55,8 +55,9 @@ public String getDescription() {
     }
 
     @Override
-    public List<Class<? extends Command>> getRelations() {
-        return Collections.singletonList(RandomCatCommand.class);
+    public List<Class<? extends Command>> getRelations()
+    {
+        return Arrays.asList(RandomCatCommand.class);
```

Comment:
What's this change for? There is no point in returning full list object when there is only one item in the list.

Reply:
It was preparations for when I get a random bird command added to AvaIre.

Reply:
Alright, at-least for this PR, the changes should be reversed to the original file so the file is not affected by a PR that is not related to it in any way.